### DISCUSSION TITLE
Export as HTML

### DISCUSSION THREAD

**Discussion by quan1997ap** [mockoon/mockoon#989] [Help] [answered]
Hi team. I like your software. Im use it for my project. But i can't export API to HTML for my project document. My customer dont want install app but he want to see all API. 
How can i do that?
Thank your support. Sorry about my english.

**Comment by 255kb** [comment, marked as answer]
Hi, there is currently no option to generate an HTML documentation. 
You can try to export as an OpenAPI spec file and then generate a swagger interface from it, using [SwaggerUI](https://swagger.io/tools/swagger-ui/) for example. 

Just be careful, import/export to OpenAPI is [not a 1:1 equivalent](https://mockoon.com/docs/latest/openapi/openapi-specification-compatibility/) to what Mockoon offers.

**Reply by quan1997ap** [reply]
Let me try it. Thank you so much
